modules/zivid/experimental/projection.py

```diff
@@ -0,0 +1,145 @@
+        will interfere with the projected image. An exception will be thrown if settings
+        contains brightness > 0 or if brightness is not set at all.
+
+        Args:
+            settings2d: A Settings2D instance to be used for 2D capture.
+
+        Returns:
+            A Frame2D containing a 2D image plus metadata.
+
+        Raises:
+            TypeError: If argument is not a Settings2D.
+        """
+
+        if isinstance(settings2d, Settings2D):
+            return Frame2D(self.__impl.capture(_to_internal_settings2d(settings2d)))
+        raise TypeError("Unsupported settings type: {}".format(type(settings2d)))
+
+    def stop(self):
+        """Stop the ongoing image projection."""
+        self.__impl.stop()
+
+    def active(self):
+        """Check if a handle is associated with an ongoing image projection.
+
+        Returns:
+            A boolean indicating projection state.
+        """
+        return self.__impl.active()
+
+    def release(self):
+        """Release the underlying resources and stop projection."""
+        try:
+            impl = self.__impl
+        except AttributeError:
+            pass
+        else:
+            impl.release()
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exception_type, exception_value, traceback):
+        self.release()
+
+    def __del__(self):
+        self.release()
+
+
+def projector_resolution(camera):
+    """Get the resolution of the internal projector in the Zivid camera.
+
+    Args:
+        camera: The Camera instance to get the projector resolution of.
+
+    Returns:
+        The resolution as a tuple (height,width).
+    """
+    return _zivid.projection.projector_resolution(
+        camera._Camera__impl  # pylint: disable=protected-access
+    )
+
+
+def show_image(camera, image_bgra):
+    """Display a 2D color image using the projector.
+
+    This function returns a ProjectedImage instance. Projection will continue until this object is
+    destroyed or if its stop() method is called. Alternatively it can be used as a context manager,
+    in which case the projection will stop when exiting the "with" block.
+
+    Args:
+        camera: The Camera instance to project with.
+        image_bgra: The image to project in the form of a HxWx4 numpy array with BGRA colors.
+
+    Returns:
+        A handle in the form of a ProjectedImage instance.
+    """
+
+    return ProjectedImage(
+        _zivid.projection.show_image(
+            camera._Camera__impl,  # pylint: disable=protected-access
+            image_bgra,
+        )
+    )
+
+
+def pixels_from_3d_points(camera, points):
+    """Get 2D projector pixel coordinates corresponding to 3D points relative to the camera.
+
+    This function takes 3D points in the camera's reference frame and converts them to the
+    projector's 2D (XY) frame using the internal calibration of a Zivid camera. The returned
+    points maps to projector pixels as round(X)->col and round(Y)->row.
```

Comment:
I think it is strange that the text here is quite different than what we have in C++. Fex the C++ text goes into more detail about calibration, while this text taks abound rounding etc. Is there a reason for it to be different? I think they should be identical except that C++ specific things must of course be replaced with Python.

Reply:
I think it happened because I didn't like the C++ docstring very much.
Anyway, I agree consistency matters also. I'll change the main body to be identical:
```
This function takes 3D points in the camera's reference frame and converts them to the projector frame
using the internal calibration of a Zivid camera. In a Zivid point cloud, each point corresponds to a
pixel coordinate in the camera, but the projector has a slight offset. The translation of each point
depends on the distance between the camera and the point, as well as the distance and angle between the
camera and the projector.
```

Regarding this stuff:
```
This function takes 3D points in the camera's reference frame and converts them to the
projector's 2D (XY) frame using the internal calibration of a Zivid camera. The returned
points maps to projector pixels as round(X)->col and round(Y)->row.
```
it's something I added because it is very easy to get confused when we return (X,Y) but then index images as (row, col). Want me to remove it?

Reply:
Fixup that makes it consistent with C++:
https://github.com/zivid/zivid-python/pull/234/commits/e39d99e1732dabd9492395aba807f31f9f9bea0c

Reply:
You can consider then to make a PR with your improvements to the C++ doc, to make it consistent? The main point is to make them consistent, but it can also be that C++ docstring can be improved.

Reply:
Let's merge this with consistency then (current fixup). Then we can always improve both for next release.

Reply:
Yes, they wont be identical since the Python specific stuff must be included as you say, so we should just aim at making them as similar as they can be, but still correct for each language. And then when we add C# documentation it will be identical to C++ and Python except for using C# specific terms/namings.

Will you make a PR to sdk master after merging this, fixing the inconsistencies then? It is fine that it is out of sync until 2.11 but I think best to fix now else we will forget.